docs/openshift.asciidoc

```diff
@@ -0,0 +1,140 @@
+[id="{p}-openshift"]
+== Openshift
+
+This section will help you run ECK on Openshift.
+
+NOTE: Only Elasticsearch and Kibana are supported on Openshift.
+
+=== Prerequisites 
+
+. The following instructions must be run as the `system:admin` user or as a `User` with some privileges to create  `Projects`, `CRDs` and `RBACs` resources at the cluster level.
+
+. Set virtual memory settings on the nodes
++
+Before deploying an Elasticsearch cluster with ECK you must ensure that the `vm.max_map_count` setting is correctly applied on all the nodes of your cluster. This is because the Pods created by ECK are likely to run with the `restricted` SCC, they will run with a limited set of privileges and will not be able to tune this setting. You can refer to the following https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html[documentation] for more information.
+
+=== Deploy the operator
+. Apply the all in one template as described in the quickstart
++
+[source,shell]
+----
+oc apply -f https://download.elastic.co/downloads/eck/0.9.0/all-in-one.yaml
+----
+
+. [Optional] If the Software Defined Network is configured with the `ovs-multitenant` plug-in you need to allow the `elastic-system` namespace to access other pods and services in the cluster: 
++
+[source,shell]
+----
+oc adm pod-network make-projects-global elastic-system
+----
+
+. Create a namespace that will hold the Elastic resources (Elasticsearch, Kibana):
```

Comment:
-> that holds
avoid Future tense wherever possible

Reply:
In this case it does not hold the resources yet though, so it seems fine? Though I might also suggest "to hold the Elastic..."